### PR TITLE
Fix Tutorial build

### PR DESCRIPTION
**Pull request by jarinosuke**
## What happens

There is an error which is missing Swift 3 migration.
This PR fixes that error on build.

I am thinking this kind of small errors doesn't need Github issues.
If it's required, please let me know, thanks.

### CODE REVIEW

**Comment by jarinosuke** [comment]
I've fixed all tutorials build

**Comment by JakeSc** [comment]
Thanks!

File: ios/tutorials/tutorial4-completed/TicTacToe/AppStart/AppDelegate.swift [hunk 1]
```diff
@@ -39,7 +39,7 @@ public class AppDelegate: UIResponder, UIApplicationDelegate {
         let result = RootBuilder(dependency: AppComponent()).build()
         launchRouter = result.launchRouter
         urlHandler = result.urlHandler
-        launchRouter?.launchFromWindow(window)
+        launchRouter.launch(from: window)
```
Comment (sbarow) [inline ios/tutorials/tutorial4-completed/TicTacToe/AppStart/AppDelegate.swift]: @jarinosuke looks like the `?` was removed.

Comment (jarinosuke) [reply ios/tutorials/tutorial4-completed/TicTacToe/AppStart/AppDelegate.swift]: @sbarow  I'm not sure why this used to be declared as optional, but it looks OK as it is.
screen shot 2017-11-28 at 18 37 35

**Comment by sbarow** [comment]
Fixed it with this https://github.com/uber/RIBs/pull/169 I think you may have pulled the fixed version :-)

**Comment by jarinosuke** [comment]
@sbarow Thanks for your help 👍

**Pull request merged**